File: Taller de Haskell/Problema 8/problema8.py
# Se pide ingresar la edad al cliente
edad = int(input("Ingrese la edad del cliente: "))

if edad <= 4:
    precio = 0
elif edad <= 12:
    precio = 5000
elif edad <= 17:
    precio = 8000
else:
    precio = 10000

print("El precio del ticket es: $", precio, "COP")

File: Taller de Haskell/Problema 8/problema8.hs
-- Se evaluan las condiciones y la edad del cliente
precioArcade :: Int -> Int
precioArcade edad
    | edad <= 4 = 0
    | edad <= 12 = 5000
    | edad <= 17 = 8000
    | otherwise = 10000

main :: IO ()
main = do
    putStrLn "Ingrese la edad del cliente:"
    entrada <- getLine
    let edad = read entrada :: Int
    putStrLn ("El precio del ticket es: $" ++ show (precioArcade edad) ++ " COP")
